hip/solver/batch_cg_launch.hip.hpp

```diff
@@ -0,0 +1,77 @@
+// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef GKO_CUDA_SOLVER_BATCH_CG_LAUNCH_HPP_
+#define GKO_CUDA_SOLVER_BATCH_CG_LAUNCH_HPP_
```

Comment:
```suggestion
#ifndef GKO_HIP_SOLVER_BATCH_CG_LAUNCH_HPP_
#define GKO_HIP_SOLVER_BATCH_CG_LAUNCH_HPP_
```

Reply:
we really should switch to `#pragma once`. This is quite annoying

Reply:
I agree.